src/js/components/Calendar/__tests__/Calendar-test.js

```diff
@@ -303,7 +303,7 @@ describe('Calendar', () => {
       </Grommet>,
     );
     // select date greater than January 1st
-    fireEvent.click(getByLabelText('Fri Jan 03 2020'));
+    fireEvent.click(getAllByLabelText('Fri Jan 03 2020')[0]);
```

Comment:
why do the tests need to use `getAllByLabelText` now?

Reply:
this was an old change i had pushed up by accident, had it changed when i was using tab Index on the day